ui/src/plugins/Notify.json

```diff
@@ -350,6 +347,7 @@
         "params": {
           "props": {
             "type": "Object",
+            "required": false,
```

Comment:
Necessary or left-over?

Reply:
There's a few more `"required": false` in the diff.

Reply:
Actually, neither. It's for increasing the clarity. `Notify.create`'s parameter is required, but the parameter of the returned method is optional. And let's say, for `from` and `to` in `QDate`, their own child properties and sibling parameters are marked as `"required": true`, so I used it explicitly to make things clearer.